pkg/action/install.go

```diff
@@ -68,6 +68,7 @@ type Install struct {
 	ChartPathOptions
 
 	ClientOnly       bool
+	NoValidate       bool
```

Comment:
This should be `Validate `for readability that that seemed to require me to update many places that initializes `Install` objects.